### ISSUE TITLE
Implement a stand-alone F3 observer node

### ISSUE THREAD

**Issue by anorth** [filecoin-project/go-f3#13]
It should be possible for a node to observe the F3 protocol without participating, and without following the EC blockchain. This would be useful for monitoring, dashboards, light clients etc, as well as an obvious demonstration of the loose coupling in protocol and code to EC/Lotus.

- [x] https://github.com/filecoin-project/go-f3/issues/94
- [x] #248 
- [x] #14 
- [x] #298

**Comment by jennijuju** [comment]
missing estimation for things after #94

**Comment by Kubuxu** [comment]
This is a combination of the already existing epic and the finishing of the standalone node. The tasks in here are covered already covered in other issues.

**Comment by jennijuju** [comment]
Can you replace the tasks with the issues if they exist then?

**Comment by Kubuxu** [comment]
done

**Comment by Kubuxu** [comment]
I think the standalone node has served its purpose for the scope of M0. I'm moving the rest of the work on standalone to M1.

**Comment by Kubuxu** [comment]
See: https://github.com/filecoin-project/go-f3/issues/319#issuecomment-2176954741 for a comment regarding standalone operation.

**Comment by rjan90** [comment]
Can we close this ticket in favour of #535?
